crate/kmip/src/kmip/kmip_types.rs

```diff
@@ -849,22 +850,6 @@ pub struct Attributes {
 }
 
 impl Attributes {
-    #[must_use]
```

Comment:
we could have keep this constructor. Why do you want to remove it?

Reply:
Because it's an empty `new()` defaulting to all `None` that can be auto-generated by deriving the `Default` trait